### PR TITLE
Hit Api

### PR DESCRIPTION
**Pull request by eamador1**
Since the last code review, the following changes were made in the project:
___

- Create a new game with the name of your choice by using the API ✅
- Implement the "Refresh" button that will get all scores for a game created by the API ✅
- Implement the form "Submit" button that will save a score for a game ✅
- Use arrow functions instead of the function keyword ✅
- Use async and await JavaScript features to consume the API ✅

### CODE REVIEW

File: src/module/showList.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export default function populateList(obj) {
+  const displayList = document.querySelector('.displayList');
```
Comment (taldr27) [inline src/module/showList.js]: - [x] Could you please update your function to use ES6 arrow functions in line with [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md)? 😊

   image

File: src/index.js [hunk 2]
```diff
@@ -1 +1,56 @@
...
+
+async function populate() {
+  const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/IV1251GkSsG9dGnKrPGk/scores/';
+  const request = new Request(requestURL);
+
+  const response = await fetch(request);
+  const allPlayers = await response.json();
+
+  populateList(allPlayers);
+}
```
Comment (taldr27) [inline src/index.js]: - [x] Could you please update your function to use ES6 arrow functions in line with [the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md)? 😊

   image

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello @eamador1, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional Readme. 
✅ Linters passing.
✅ Correct use of GitHub Flow.
✅ Good job fetching and posting your scores.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Comment by eamador1** [comment]
Since the last code review, the following changes were made:
___

- Update two functions to ES6 arrow functions ✅

**Review by ademibowale — APPROVED** [review]
Hi @eamador1  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️
- Use arrow functions ✔️
- The submit button should hit the POST API endpoint and post user entries✔️
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**